benefits/core/templates/core/base.html

```diff
@@ -7,9 +7,13 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="description" content="{% translate "core.pages.help.about.p[0]" %}">
+    {% blocktranslate trimmed asvar description %}
```

Comment:
`asvar` is the syntax to store the results of a `{% blocktranslate %}` into a variable: https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#std-templatetag-blocktranslate

Reply:
`trimmed` removes extra whitespace in the translated block https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#std-templatetag-blocktranslate